careers/software-engineer.md

```diff
@@ -44,6 +73,21 @@ An engineer II is capable of taking on most technical problems and coordinating
 
 **Collaboration:** They are mindful of the work that others are doing and can critique or suggest improvements to the proposed plans of others. They get help from others when stuck, but are also capable of helping unstick their peers. They are capable of effectively running small to medium-sized projects with regular coaching.
 
+### Responsibilities
+
+* Design and develop software components or features with moderate complexity.
+* Write clean, efficient, and well-documented code with less direct supervision.
+* Perform more thorough software testing, including unit and integration tests.
+* Contribute to technical specifications and requirements.
+* Troubleshoot and debug more complex software issues and identify root causes.
+* Participate in code reviews and provide constructive feedback including opportunities to improve code quality.
+* Work independently on medium-sized tasks.
```

Comment:
This doesn't add much, IMO. Maybe we axe this or at least write something closer to what the "Impact" section articulates?

> Lead implementation of features requiring multiple components or spanning several sprints, including planning and breaking down the work